spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -617,50 +663,17 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
           val value = cast.eval()
           exprToProtoInternal(Literal(value, dataType), inputs)
 
+        case UnaryExpression(child) if expr.prettyName == "trycast" =>
```

Comment:
Just making sure whether it is `trycast`, not `try_cast`. Looks like Spark 3.4 is using `try_cast`, haven't been able to find the source for Spark 3.3